Q/QPALM/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QPALM"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Benny44/QPALM_vLADEL.git", "370ccbeb33baaa566c3de2bde5a69944994593e2"),
+    GitSource("https://github.com/Benny44/LADEL.git", "585b766da25e028f368e4d9039ef69e326baeb5b"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd QPALM_vLADEL/
+ln -s ${WORKSPACE}/srcdir/LADEL/* ${WORKSPACE}/srcdir/QPALM_vLADEL/LADEL/
+[[ $target == *-freebsd* ]] && patch -p1 < $WORKSPACE/srcdir/patches/freebsd.patch
+mkdir -p build
+cd build
+mkdir lib debug
+cd debug  # just following the instructions here
+
+[[ $target == *-mingw* ]] && BLAS=libopenblas.${dlext}.a || BLAS=libopenblas.${dlext}
+[[ $target == *-linux-* ]] && std="gnu99" || std="c99"
+
+cmake ../.. \
+      -DCMAKE_BUILD_TYPE=release \
+      -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DUSE_LADEL=ON \
+      -DUSE_CHOLMOD=ON \
+      -DINTERFACES=OFF \
+      -DPYTHON=OFF \
+      -DCOVERAGE=OFF \
+      -DUNITTESTS=OFF \
+      -DJULIA=OFF \
+      -DLAPACKE=${WORKSPACE}/destdir/lib/${BLAS} \
+      -DLAPACK=${WORKSPACE}/destdir/lib/${BLAS} \
+      -DCMAKE_C_FLAGS="-I${WORKSPACE}/destdir/include -std=${std}"
+
+make
+
+# make install doesn't appear to work...
+if [[ $target == *-mingw* ]]; then
+  # headers don't make it to build/include
+  cp ../../include/* $prefix/include
+  cp bin/* $prefix/bin
+else
+  cp ../include/* $prefix/include
+  cp ../lib/* $prefix/lib
+fi
```

Comment:
Without conditionals:
```suggestion
# headers don't make it to build/include
cp ../../include/* ${includedir}
cp $(basename ${libdir})/* ${libdir}
```

Reply:
No, sorry, this doesn't work. I'll undo this.